portal/src/main/webapp/WEB-INF/jsp/global/selected_study_variables.jsp

```diff
@@ -21,9 +21,8 @@
 %>
 <script type="text/javascript">
 
-    var isVirtualStudy = <%=isVirtualStudy%>;
-    var cancerStudyIdList = '<%=cancerStudyIdListStr%>'; // empty string if single study
-    var cancerStudyId = '<%=cancerStudyId%>'; // if multi-studies, this is always "all"
-    window.cohortIdsList = (cancerStudyIdList === 'null')? [cancerStudyId]: cancerStudyIdList.split(',');
+    window.cancerStudyIdList = '<%=cancerStudyIdListStr%>'; // empty string if single study
```

Comment:
Is this file used on multiple pages?  We just need to make sure this doesn't break them.  We had a bug recently having to do this this.